app/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager/cloud_subnet.rb

```diff
@@ -5,6 +5,49 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::NetworkManager::CloudS
     end
   end
 
+  def self.params_for_create(_ems)
+    {
+      :fields => [
+        {
+          :component => 'select',
+          :name      => 'type',
+          :id        => 'type',
+          :label     => _('Type'),
+          :options   => [
```

Comment:
@jaywcarman is this field a two-state, or can it be also empty?

Reply:
This field must be one of the two states.

Reply:
Okay, in that case it should be required...changing